feed-ingress/cmd/alb.go

```diff
@@ -20,6 +20,15 @@ Specifically, they don't respect the deregistration delay. As a result, we don't
 
 func init() {
 	rootCmd.AddCommand(albCmd)
+
+	elbCmd.Flags().StringVar(&region, "region", defaultRegion,
+		"AWS region for frontend attachment.")
+	elbCmd.Flags().StringSliceVar(&targetGroupNames, "alb-target-group-names", []string{},
+		"Names of ALB target groups to attach to, separated by commas.")
+	elbCmd.Flags().DurationVar(&targetGroupDeregistrationDelay, "alb-target-group-deregistration-delay",
+		defaultTargetGroupDeregistrationDelay,
+		"Delay to wait for feed-ingress to deregister from the ALB target group on shutdown. Should match"+
+			" the target group setting in AWS.")
```

Comment:
they are still being added to the elbCmd :(

Reply:
D'oh!